pkg/grizzly/workflow.go

```diff
@@ -220,40 +225,48 @@ func Apply(registry Registry, resources Resources) error {
 	for _, resource := range resources {
 		handler, err := registry.GetHandler(resource.Kind())
 		if err != nil {
+			// Why?
```

Comment:
Fair question. We should return a "handler not found for this Kind" type error.

Reply:
I'll do that, thanks for the suggestion. Also, I totally did not intend to commit these 😆

Reply:
Done.